app/controllers/employees_controller.rb

```diff
@@ -33,6 +35,7 @@ def update
     @employee = Employee.find(params[:id])
     @employee.slack_username = params[:employee][:slack_username]
     @employee.validate_slack_username_in_org
+    @employee.add_slack_user_id_to_employee
```

Comment:
wondering if we should move some of this logic to its own class. Feels like the controller is getting bigger and bigger. Doesn't need to happen now, but if you were up for it that would be cool.